app/controllers/document_lead_image_controller.rb

```diff
@@ -38,12 +38,12 @@ def create
 
   def edit
     @document = Document.find_by_param(params[:document_id])
-    @image = Image.find_by(id: params[:image_id])
+    @image = Image.find(params[:image_id])
   end
 
   def update
     document = Document.find_by_param(params[:document_id])
-    image = Image.find_by(id: params[:image_id])
+    image = Image.find(params[:image_id])
```

Comment:
Similarly on these probably should be `document.images.find` if we want to strictly match the url